targets.config.js

```diff
@@ -104,8 +109,9 @@ module.exports = {
             options: {
                 ...publicOptions,
                 ...icons.production,
-                extensionFullName: 'Accessibility Insights for Web - Preview',
+                fullName: 'Accessibility Insights for Web - Preview',
```

Comment:
I guess we should remove **Preview** since we are not actually publishing it but I can see that being a different PR

Reply:
Yup. I can take care of that after this is merged!